app/gui2/src/components/ComponentBrowser/input.ts

```diff
@@ -81,58 +80,72 @@ export function useComponentBrowserInput(
   suggestionDb: SuggestionDb = useSuggestionDbStore().entries,
   ai: { query(query: string, sourcePort: string): Promise<Result<string>> } = useAI(),
 ) {
-  const code = ref('')
+  const text = ref('')
   const cbUsage = ref<Usage>()
   const anyChange = ref(false)
   const selection = ref({ start: 0, end: 0 })
-  const ast = computed(() => RawAstExtended.parse(code.value))
+  const ast = computed(() => RawAstExtended.parse(text.value))
   const imports = ref<RequiredImport[]>([])
   const processingAIPrompt = ref(false)
   const toastError = useToast.error()
+  const sourceNode = ref<string>()
```

Comment:
This is an identifier, I guess? Then please put that in the name (string could be theoretically ExpressionId, node's expression or whatever).